src/backend/expungeservice/crawler/models/case.py

```diff
@@ -12,4 +12,13 @@ def __init__(self, info, case_number, citation_number, date_location, type_statu
         self.date = datetime.date(datetime.strptime(date, '%m/%d/%Y'))
         self.violation_type, self.current_status = type_status
         self.charges = charges
+        self.balance_due_in_cents = 0
```

Comment:
This attribute name was a bit confusing at first as I'm used to seeing the getter/setters the same name as the attribute, but after a bit I understand the thought process here.